populate_dev_data_handler.go

```diff
@@ -23,42 +23,49 @@ import (
     "appengine/datastore"
 )
 
+// Create TestRun entities for local development and testing.
+// These point to JSON files stored in /static/.
+// Key names for each entity are specified to make this route idempotent.
 func populateDevData(w http.ResponseWriter, r *http.Request) {
     ctx := appengine.NewContext(r)
```

Comment:
Question, unrelated to this PR: Do you know what purpose these Context structs have? Like, would you ever want to make more than one "Context" for a Request?